### ISSUE TITLE
Storybook Canvas (demos): Accessibility issues with incorrectly attributed `disabled` attribute

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2045]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

The following components demos (Canvas tab) have `disabled="false"` on by default - this should be removed altogether. As it stands all buttons are listed as disabled and are not accessible. This needs to be fixed.

- modus-button
- modus-chip

E.g. On the page: https://modus-web-components.trimble.com/?path=/story/components-button--default the Sentiment scale attribute has:
`<modus-button disabled="false">`

The disabled attribute should be removed otherwise it will be announced as unavailable (disabled)
![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/49b8dcb5-714b-457a-ab8b-5438a7076c4c)
